svm30/svm30.c

```diff
@@ -54,11 +54,16 @@ static uint32_t sensirion_calc_absolute_humidity(const int32_t *temperature,
                                                  const int32_t *humidity) {
```

Comment:
Should we make it a non static function so that it is easier to test it? Otherwise the test would need to include the `.c` file to have access to the function.

Reply:
I would advise against optimizing for testing. We easily can find solutions to that problem (even if somewhat hacky), but code-customers will have a harder time figuring out what can be optimized to shed a couple of bytes from the resulting binary.